src/articles/playbooks/guides/sharing-usage-report.md

```diff
@@ -0,0 +1,108 @@
+
+## Add a Trigger
+
+The first step is to add a Schedule trigger. Click "Add a trigger" on the canvas to bring up the "Add Triggers" panel. From there you can type "Schedule" in the filter to find the "Schedule Trigger".
+
+<img src="/public/images/articles/playbooks.guides.sharing-usage-report/add-trigger-panel.png">
+
+Click on the Schedule trigger to choose and configure it. In the screenshot below we're sticking with the defaults to run this daily at midnight in my timezone.
+
+<img src="/public/images/articles/playbooks.guides.sharing-usage-report/configure-schedule.png">
+
+Click "Save" to save the step. Don't worry, nothing will happen until we publish the Playbook.
+
+After adding the trigger, you'll notice a Dispatching option. Select "Run once for each customer". This provides the option run this playbook for a specific set of customer segments. For more about setting up customers and customer segments, visit [customers setup](../../customers/setup.md). In the screenshot below you can see this Playbook will only send the report to our esteemed Business and Enterprise customers.
+
+<img src="/public/images/articles/playbooks.guides.sharing-usage-report/dispatching.png">
+
+## Filtering Out Customers Without Rooms
+
+This Playbook relies on having set up customers in Abbot and associating them with a Room (Slack channel). We want to filter out customers who do not have a channel associated with them.
+
+> [!TIP]
+> If you have a lot of Rooms in Abbot not associated with customers, visit the [Create Customers for Rooms](https://app.ab.bot/settings/rooms/createcustomers) page. You'll see a list of all customers without rooms and have the option to create customer records for each room.
+
+To do so, we'll add a "Continue If" step. Click "Add an action" and then filter the list to find the "Continue If" step. Click on the step to add and configure it.
+
+<img src="/public/images/articles/playbooks.guides.sharing-usage-report/continue-if-step.png">
+
+Select the "Channel from Trigger" condition and the "Exists" comparison. This ensurse the playbook will only continue if the customer the playbook is running for has an associated channel.
+
+> [!NOTE]
+> If you don't see any options, make sure "Run once for each customer" is selected in the dispatch options.
+
+<img src="/public/images/articles/playbooks.guides.sharing-usage-report/playbook-with-two-steps.png">
+
+## Gather Stats for the Report
+
+In this step, we need to gather your customer usage stats to send in the report. To do that, we need to run a skill. So the first step is to write a skill that calls out to an API you control to get these stats. To create a skill, click on "Skills" in the left navigation and click "Create new skill". In this example, we'll create a C# skill. To jump to the create page, [click here](https://app.ab.bot/skills/create/csharp). Give the skill a name, click "Create Skill", and then enter the following code in the Skill editor:
+
+```csharp
```

Comment:
This is ideal for some [Tabs](https://dotnet.github.io/docfx/docs/markdown.html?tabs=linux%2Cdotnet#tabs). I also wonder if we should use JavaScript instead of C# 🤔 , but I'm not going to have you rewrite the thing now. We can add JS later and make it the default tab.

Reply:
Unfortunately the customer API is not available for JS skills yet.